src/v/cluster/partition_probe.h

```diff
@@ -56,6 +57,10 @@ class partition_probe {
         return _impl->add_bytes_produced(bytes);
     }
 
+    void add_batches_produced(uint64_t bytes) {
+        return _impl->add_batches_produced(bytes);
```

Comment:
nit: i guess you probably meant to use a name other than `bytes`?

Reply:
Thanks, yes of course. Fixed